rafthttp/msgapp.go

```diff
@@ -35,11 +35,15 @@ type msgAppEncoder struct {
 }
 
 func (enc *msgAppEncoder) encode(m raftpb.Message) error {
+	start := time.Now()
```

Comment:
the start time is before the length writing procedure here, which is different from other encode and decode.
What is the plan for encoding time? Does it include message-length write time?